pyproject.toml

```diff
@@ -34,7 +34,8 @@ classifiers = [
 ]
 
 [project.optional-dependencies]
-docs = [ "sphinx>=7"]
+docs = ["sphinx>=7"]
```

Comment:
I'd prefer to leave that line alone so it doesn't get blamed back to an unrelated patch.

Reply:
done